examples/setmodescale.py

```diff
@@ -17,7 +17,7 @@
 FPS = 30
 clock = pg.time.Clock()
 
-print ("desktops", pg.display.get_desktop_sizes())
+print("desktops", pg.display.get_desktop_sizes())
```

Comment:
Sorry I just got around to this @Starbuck5. This line breaks py2.7 compat. Are we officially dropping it?

Reply:
I'll put in a PR that fixes this by doing `from __future__ import print_function`

Reply:
Yeah, that would fix it, too. This was the one thing I noticed going through the diffs that I told you about. Not a blocker though :)